companies/holvi.json

```diff
@@ -0,0 +1,27 @@
+{
+    "slug": "holvi",
+    "relevant-countries": [
+        "at",
+        "de",
+        "fi"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Holvi Payment Services Oy",
+    "address": "Hämeentie 11\n00530 Helsinki\nFinland",
+    "phone": "+358 75 325 2935",
+    "fax": "+358 92 319 4337",
+    "email": "privacy@holvi.com",
+    "web": "https://www.holvi.com/",
+    "sources": [
+        "https://support.holvi.com/hc/en-gb/articles/202260402-Privacy-Policy-Personal-Data-Declaration"
+    ],
+    "comments": [
```

Comment:
Please remove the comments, the relevant information is already in the record.

Reply:
`comments` have been removed.